docs/create/kubernetes/kubernetes-operator/index-kubernetes-operator.mdx

```diff
@@ -0,0 +1,380 @@
+
+Run the below command to create the Deployment resource:
+
+
+```
+kubectl create -f nginx-dep.yaml
+```
+
+
+
+
+
+
+
+
+Let us verify if the Deployment was created successfully by running the following command:
+
+
+```
+ kubectl get deployments
+ NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+ nginx-deployment   2/2     2            2           63s
+```
+
+
+The example above shows the name of the Deployment in the namespace. It also displays how many replicas of the application are available to your users. You can also see that the number of desired replicas that have been updated to achieve the desired state is 2. 
+
+![alt_text](images/image2.png "image_tooltip")
+
+You can run the “kubectl describe” command to get detailed information of deployment resources. Show details of a specific resource or group of resources
+
+
+```
+ kubectl describe deploy
+ Name:                   nginx-deployment
+ Namespace:              default
+ CreationTimestamp:      Mon, 30 Dec 2019 07:10:33 +0000
+ Labels:                 <none>
+ Annotations:            deployment.kubernetes.io/revision: 1
+ Selector:               app=nginx
+ Replicas:               2 desired | 2 updated | 2 total | 0 available | 2 unavailable
+ StrategyType:           RollingUpdate
+ MinReadySeconds:        0
+ RollingUpdateStrategy:  25% max unavailable, 25% max surge
+ Pod Template:
+   Labels:  app=nginx
+   Containers:
+    nginx:
+     Image:        nginx:1.7.9
+     Port:         80/TCP
+     Host Port:    0/TCP
+     Environment:  <none>
+     Mounts:       <none>
+   Volumes:        <none>
+ Conditions:
+   Type           Status  Reason
+   ----           ------  ------
+   Available      False   MinimumReplicasUnavailable
+   Progressing    True    ReplicaSetUpdated
+ OldReplicaSets:  <none>
+ NewReplicaSet:   nginx-deployment-6dd86d77d (2/2 replicas created)
+ Events:
+   Type    Reason             Age   From                   Message
+   ----    ------             ----  ----                   -------
+   Normal  ScalingReplicaSet  90s   deployment-controller  Scaled up replica set nginx-deployment-6dd86d77d to 2
+```
+
+
+A Deployment is responsible for keeping a set of Pods running, but it’s equally important to expose an interface to these Pods so that the other external processes can access it. That’s where the Service resource comes in. The Service resource lets you expose an application running in Pods to be reachable from outside your cluster. Let us create Service resource definition as shown below:
```

Comment:
```suggestion
A Deployment is responsible for keeping a set of Pods running, but it’s equally important to expose an interface to these Pods so that the other external processes can access them. That’s where the Service resource comes in. The Service resource lets you expose an application running in Pods to be reachable from outside your cluster. Let us create a Service resource definition as shown below:
```